### DISCUSSION TITLE
Feeding OGN using balena and Docker

### DISCUSSION THREAD

**Discussion by ketilmo** [VirusPilot/ogn-pi34#4] [General]
Hi @VirusPilot,

Thanks for creating this excellent repo! I'm the maintainer of [balena-ads-b](https://github.com/ketilmo/balena-ads-b), a solution for feeding ADS-B data to the most popular flight tracking solutions using Docker and balena orchestration.

As a paraglider pilot myself, I wanted to create a similar solution for OGN, that could also run side-by-side with the ADS-B feeding. Over the past few years, I've tried to get the default OGN setup working in the Docker and balena environments. While I was partially successful, I could not get it to run on x64-based operating systems, which is required when using balena on RPi 4. But to my delight, I got it to work using your distribution. So thanks a lot for providing this. Great work!

I want to make the modifications available to the community and ask your permission to use your distribution files and scripts as part of this setup. I will, of course, credit you and link back to the original project.

**Comment by VirusPilot** [comment]
Hi @ketilmo ,

Sure, I am happy to hear that my scripts are useful for your project. Please feel free to use my files and scripts as part of your setup!

**Reply by ketilmo** [reply]
Thank you so much, @VirusPilot! I appreciate it! I will report back here when I have everything ready.

**Comment by VirusPilot** [comment]
Looking forward to your next release!

**Comment by ketilmo** [comment]
Hi @VirusPilot, 

It's been a while, but the balena OGN setup is finally up and running. I've been testing and incrementally adjusting it during the past few months at several sites, and it's working well. Now I need to finish the documentation so others can make use of this, too.

You can have a look here: https://github.com/ketilmo/balena-ogn

Thank you so much for your excellent work in this area and your assistance!

**Comment by VirusPilot** [comment]
Sure, happy to see that it works fine. The OGN libraries have recently beed upgraded to v0.3.2 to ensure compatibility with the latest FLARM update.

**Reply by ketilmo** [reply]
Thank you! Yes, I updated to v0.3.2 earlier this week. It seems to be working well.

**Comment by VirusPilot** [comment]
One minor thing - I suggest removing the following from "ogn.conf.tpl" as this was only related to a private project from Pawel who is the main developer behind OGN:

FRB:
{
  DetectSNR = 10.0;
  Server = "ogn3.glidernet.org:50000";
};

Furthermore please note some more comments in the "Template.conf", e.g. setting the default GSM frequency to "0".

**Reply by ketilmo** [reply]
Thanks for pointing this out, @VirusPilot! It's appreciated. I have now removed the FRB section from the config, and changed the GSM center frequency and gain according to the values you have in your setup.

**Reply by VirusPilot** [reply]
Sure, no problem.

**Comment by ketilmo** [comment]
Hi again,

I have now created the [docs](https://github.com/ketilmo/balena-ogn) and deployed the solution to the [balena Hub](https://hub.balena.io/apps/2130406/balena-ogn) for easy deployment. Thanks again for your assistance and great work!

**Comment by VirusPilot** [comment]
Thank you for considering my work!

**Comment by PA5MN-5** [comment]
Is there comprehensive installation help for non-programmers? I've tried several different installations and versions, but without success.
I have two Pi 3s working with FLARM and ADS B, and I want to get it working with a Pi 4 and provide FLARM and ADS B data to Glider Radar.
Thanks for the great work!

**Comment by VirusPilot** [comment]
I suggest that you consider my docker version: [docker-ogn2readsb](https://github.com/VirusPilot/docker-ogn2readsb.) This is simple to install and even provides you with a unified map of the traffic around your station.

**Comment by PA5MN-5** [comment]
Installed docker-ogn2readsb and works perfectly!
Thanks for the hard work!

PA5MN
Mario L. Nienhuis
Solkuillaan 62
3903DA Veenendaal
The Netherlands

Mob : +31(0)621-564528


* * * D I S C L A I M E R * * *

The information contained in this e-mail is confidential and may be
privileged. It may be read, copied and used only by the intended recipient.
If you have received it in error, please contact the sender immediately by
return e-mail. Please delete this e-mail and do not disclose its contents
to any person. Sender does not accept any liability for any errors,
omissions, delays of receipt or viruses in the contents of this message
which arise as a result of e-mail transmission.
* * * * * * * * * * * * * * * *



Op wo 17 dec 2025, 18:17 schreef VirusPilot ***@***.***>:

**Comment by VirusPilot** [comment]
Thank you for your feedback, much appreciated!
